routing/bicycle_directions.cpp

```diff
@@ -385,7 +385,7 @@ void BicycleDirectionsEngine::FillPathSegmentsAndAdjacentEdgesMap(
     // It leads to preventing pushing item to |prevSegments|. So if there's no enough items in |prevSegments|
     // |pathSegment.m_trafficSegs| should be empty.
     // Note. For the time being BicycleDirectionsEngine is used for turn generation for bicycle and car routes.
-    if (m_generateTrafficSegs && prevSegments.size() + 1 == prevJunctionSize)
+    if (prevSegments.size() + 1 == prevJunctionSize)
```

Comment:
Am I right that this is always true now? Am I right that prevSegments.size() is always 1 at this point?